cli/lib/kontena/cli/master/aws/create_command.rb

```diff
@@ -19,6 +19,8 @@ class CreateCommand < Clamp::Command
     option "--mongodb-uri", "URI", "External MongoDB uri (optional)"
     option "--version", "VERSION", "Define installed Kontena version", default: 'latest'
     option "--auth-provider-url", "AUTH_PROVIDER_URL", "Define authentication provider url (optional)"
+    option "--associate_public_ip_address", :flag, "Whether to associated public IP in case the VPC defaults to not doing it", default: false, attribute_name: :associate_public_ip
```

Comment:
I would change this to hyphened format: `--associate-public-ip-address`, like the rest of the options.